File: Python/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris):
    def __init__(self, id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna, hewan, size, merk):
        super().__init__(id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna)
        self.hewan = hewan
        self.size = size
        self.merk = merk

File: Java/Baju.java
// Child dari Aksesoris
public class Baju extends Aksesoris {
    private String hewan, size, merk;

    public Baju(int id, String nama, double harga, int stok, String jenis, String bahan, String warna, String hewan, String size, String merk) {
        super(id, nama, harga, stok, jenis, bahan, warna);
        this.hewan = hewan;
        this.size = size;
        this.merk = merk;
    }

    @Override
    public void tampilkanData() {
        System.out.printf("| %-3d | %-16s | %-10.2f | %-4d | %-9s | %-7s | %-6s | %-6s | %-4s | %-10s |\n",
                id, nama, harga, stok, jenis, bahan, warna, hewan, size, merk);
    }
}